airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -422,3 +422,50 @@ def create_or_update_glue_job(self) -> str | None:
             self.conn.create_job(**config)
 
         return self.job_name
+
+
+class GlueDataBrewHook(AwsBaseHook):
+    """
+    Interact with AWS DataBrew.
+
+    Additional arguments (such as ``aws_conn_id``) may be specified and
+    are passed down to the underlying AwsBaseHook.
+
+    .. seealso::
+        - :class:`~airflow.providers.amazon.aws.hooks.base_aws.AwsBaseHook`
+    """
+
+    def __init__(self, *args, **kwargs):
+        kwargs["client_type"] = "databrew"
+        super().__init__(*args, **kwargs)
+
+    def job_completion(self, job_name: str, run_id: str, delay: int = 10, max_attempts: int = 60) -> str:
+        """
+        Wait until Glue DataBrew job reaches terminal status.
+
+        :param job_name: The name of the job being processed during this run.
+        :param run_id: The unique identifier of the job run.
+        :param delay: Time in seconds to delay between polls
+        :param maxAttempts: Maximum number of attempts to poll for completion
+        :return: job status
+        """
+        self.get_waiter("job_complete").wait(
+            Name=job_name,
+            RunId=run_id,
+            WaiterConfig={"Delay": delay, "maxAttempts": max_attempts},
+        )
+
+        status = self.get_job_state(job_name, run_id)
+        return status
+
+    def get_job_state(self, job_name: str, run_id: str) -> str:
+        """
+        Get the status of a job run.
+
+        :param job_name: The name of the job being processed during this run.
+        :param run_id: The unique identifier of the job run.
+        :return: State of the job run.
+            'STARTING'|'RUNNING'|'STOPPING'|'STOPPED'|'SUCCEEDED'|'FAILED'|'TIMEOUT'
+        """
+        response = self.get_conn().describe_job_run(Name=job_name, RunId=run_id)
```

Comment:
(possibly elsewhere as well) Preference is to use the `conn` cached property over the old `get_conn()` helper now.
```suggestion
        response = self.conn.describe_job_run(Name=job_name, RunId=run_id)
```